pytest_fixtures/core/contenthosts.py

```diff
@@ -198,7 +198,7 @@ def katello_host_tools_tracer_host(rex_contenthost, target_sat):
 
 
 @pytest.fixture(scope='module')
-def module_container_contenthost(request, module_target_sat):
+def module_container_contenthost(request, module_target_sat, default_org):
```

Comment:
It is used regardless the Org (it does not matter which Org it is registered to), so I suppose we can go with any of them.

Reply:
if host is registered in default_org then it won't be accessible in other orgs, which might fail a most of tests
I think we'll need to create a func scoped and module scoped fixtures for same purpose

Reply:
No. The host does not need to be registered in any Org to search and pull images, it was not registered at all till now.
The only two things it needs are:
1. podman/docker client installed
2. accept Sat's cert to avoid need for `--tls-verify=false`